### ISSUE TITLE
Lower default dropout for language models

### ISSUE THREAD

**Issue by alanakbik** [zalandoresearch/flair#349]
Currently, the `LanguageModel` class sets a dropout of 0.5 by default. However, in most cases there is little danger of overfitting, since text corpora used to train LMs tend to be very large. We noted that we get improved results if we reduce the dropout to 0.1 or even no dropout at all (see discussion in #329). 

Task: Set a lower default value for dropout in the `LanguageModel`.

**Comment by alanakbik** [comment]
closed by #350
